lua/earmuffs/client/cl_ambiance.lua

```diff
@@ -0,0 +1,29 @@
+--    -- Bigcity ambiance
```

Comment:
What's the plan for this file? Do we want a separate slider for ambient sounds?

Reply:
Yeah I was thinking of having multiple sliders, or maybe one master slider with check boxes to include other "modules"